cert_issuer/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '3.1.0'
+__version__ = '3.2.0'
```

Comment:
this is weird, it should already be that version in master: https://github.com/blockchain-certificates/cert-issuer/blob/master/cert_issuer/__init__.py#L1

Reply:
ah, can you merge master branch in your working branch?

Reply:
I merged from master and I retested that it can issue on testests (refs [goerli tx](https://goerli.etherscan.io/tx/0xba9e78e90324eb80334c502395827dc076045494541587b6b61dbe49cc678cd6), [ropsten tx](https://ropsten.etherscan.io/tx/0x70b2d032b4c53bcec13d1b1aa2dc0f2521a92e621e76d4b825e2c9a044fc3886))